itests/kit/ensemble.go

```diff
@@ -208,11 +216,15 @@ func (n *Ensemble) Miner(miner *TestMiner, full *TestFullNode, opts ...NodeOpt)
 			genm    *genesis.Miner
 		)
 
+		// Default 2KiB sector for the network version
```

Comment:
nit: seems like this should be an ensemble option so the default can be overridden, though maybe this should be left to whoever first wants to use non-2KiB sectors in an itest

Reply:
Yeah, I'm going to leave it to them.